apps/web-nextjs/src/components/ui/input-otp.tsx

```
@@ -0,0 +1,79 @@
+        </div>
+      )}
+    </div>
+  );
+}
+
+function InputOTPSeparator({ ...props }: React.ComponentProps<"div">) {
+  return (
+    <div
+      data-slot='input-otp-separator'
+      role='separator'
+      {...props}>
```

Biome (2.1.2)

[error] The HTML element with the interactive role 'separator' is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like 'separator'.
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)